torch-sys/src/io.rs

```diff
@@ -0,0 +1,80 @@
+use libc::{c_void, size_t};
+use std::io::{Write, Read, Seek, SeekFrom};
+
+pub trait ReadStream : Read+Seek {}
```

Comment:
Is there a reason for adding a new trait rather than relying on the existing `Read` and `Seek` one? Having a different trait here makes it harder for user to use this on types that are not defined in their crate and that would implement both `Read` and `Seek` (they can wrap the type but this doesn't sound ideal).